### PR TITLE
Replace 'binary' with 'executable'

### PR DESCRIPTION
**Pull request by dudu84**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by dudu84** [comment]
Hi! My commit message is wrong. Also I should have done 6 commits (one for each changed file) instead of one. Someone can help me fix this?

**Comment by arjunsinghy96** [comment]
`squash`ing these commits is also an option.  But the above method is better.

**Comment by dudu84** [comment]
@arjunsinghy96 thanks so much for your help!
 I did as you explained but still got all this errors. Also it is conflicting again which I don't understand why.

**Comment by dudu84** [comment]
@arjunsinghy96 thanks so much for your help!

I did as you explained but still got all this errors. Also it is conflicting again what I don't understand why.

File: coalib/coala_ci.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from coalib.coala import main as coala_main
 
 
-def main():
-    logging.warning('Use of `coala-ci` binary is deprecated, use '
+def main(debug=False):
```
Comment (arjunsinghy96) [inline coalib/coala_ci.py]: This change is conflicting with https://github.com/coala/coala/commit/fa4abdd13e92c33b85df699c54030d0e44a75e66 . This changes are not required by your issue. Remove this change and rebase. This should remove your conflict problem.

File: coalib/coala_format.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
 from coalib.coala import main as coala_main
 
 
-def main():
-    logging.warning('Use of `coala-format` binary is deprecated, use '
+def main(debug=False):
```
Comment (arjunsinghy96) [inline coalib/coala_format.py]: This change is also conlicting with https://github.com/coala/coala/commit/fa4abdd13e92c33b85df699c54030d0e44a75e66. Do same for this.

File: coalib/coala_json.py [hunk 3]
```diff
@@ -4,8 +4,8 @@
 from coalib.coala import main as coala_main
 
 
-def main():
-    logging.warning('Use of `coala-json` binary is deprecated, use '
+def main(debug=False):
```
Comment (arjunsinghy96) [inline coalib/coala_json.py]: This also. ^^^

Comment (arjunsinghy96) [inline coalib/coala_json.py]: Change these. And rebase.

**Comment by userzimmermann** [comment]
@dudu84 You are almost there... :)

**Comment by jayvdb** [comment]
Please rebase.  See the guide in our developer documentation.

**Comment by jayvdb** [comment]
ack c305ea7

**Comment by jayvdb** [comment]
Great.  Now needs a rebase, and ping a maintainer on gitter to get it merged after the builds are green.

**Comment by dudu84** [comment]
@userzimmermann  Hi there! Could you please merge this?

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
@dudu84 Sorry, did my best to merge it ;) But it really needs a final rebase

**Comment by jayvdb** [comment]
@dudu84 , you havent pushed a rebased commit to this branch yet.

This PRs commits can be seen at https://github.com/coala/coala/pull/4288/commits

It is still c305ea7 , which is from many days ago.

Our processes are designed to ensure people know `git`, and also we use `rultor` to have a clean git history without any merge commits.  GitHub default process tries to avoid people needing to know much about git.

**Comment by dudu84** [comment]
@jayvdb Just did. Is it right now?

**Comment by jayvdb** [comment]
Now you have four commits on https://github.com/coala/coala/pull/4288/commits
c305ea7 , followed by 46d8fb9, 31d6dfb and 6828af4 .

This is not a correct rebase.  You've done an inverse rebase.  The core of the problem is that you've used "master" as your PR branch, which does make rebasing a bit harder.

If you cant figure out how to fix this PR, please come onto http://coala.io/chat and we can guide you through the process.

**Comment by jayvdb** [comment]
ack d127897

**Comment by jayvdb** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
@dudu84 🎉 :)

**Comment by dudu84** [comment]
@userzimmermann sure they will! Thanks man!

**Pull request merged**